src/workerd/server/workerd-api.h

```diff
@@ -38,7 +38,8 @@ class WorkerdApi final: public Worker::Api {
       kj::Own<JsgIsolateObserver> observer,
       api::MemoryCacheProvider& memoryCacheProvider,
       const PythonConfig& pythonConfig,
-      kj::Maybe<kj::Own<jsg::modules::ModuleRegistry>> newModuleRegistry);
+      kj::Maybe<kj::Own<jsg::modules::ModuleRegistry>> newModuleRegistry,
+      kj::Maybe<jsg::VfsCallback> vfsCallback = kj::none);
```

Comment:
```suggestion
      kj::Maybe<jsg::VfsCallback> vfsCallback);
```

Reply:
For now, until the internal repo PR is opened that provides this, I'm going to keep the default values here.